src/test/scala/org/bitcoins/core/crypto/TransactionSignatureSerializerTest.scala

```diff
@@ -221,7 +222,7 @@ class TransactionSignatureSerializerTest extends FlatSpec with MustMatchers with
      val rawTx = TestUtil.simpleRawTransaction
      val rawParentTx = TestUtil.parentSimpleRawTransaction
      val bitcoinjTx = new org.bitcoinj.core.Transaction(params,Utils.HEX.decode(rawTx))
-     val input = bitcoinjTx.getInput(inputIndex)
+     val input = bitcoinjTx.getInput(inputIndex.underlying.toInt)
```

Comment:
There is a bunch of these in this file, needs to be just `.toInt` instead of `underlying.toInt`